### DISCUSSION TITLE
[V3] Upgrade script missing @this

### DISCUSSION THREAD

**Discussion by dakira** [livewire/livewire#6159] [7. Bugs (archived)]
### Livewire version

v4

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Create a blank Jetstream livewire project
2. Upgrade to v3
3. Run upgrade script.

The upgrade script should replace occurrences of `@this.on()` with `Livewire.on()`.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
**Edit** sorry you meant `@this.on()`. Yeah that should be changed to Alpine's event syntax. @PhiloNL tagging for visibility 🙂

@dakira thanks for reporting! `@this` should still work, is it not working for you?
